vendors/circom/benchmark/circom_benchmark.cc

```diff
@@ -0,0 +1,129 @@
+#include <stddef.h>
+#include <stdint.h>
+
+#include <iostream>
+#include <utility>
+
+#include "alt_bn128.hpp"  // NOLINT(build/include_subdir)
+#include "openssl/sha.h"
+
+// clang-format off
+#include "benchmark/bit_conversion.h"
+#include "benchmark/tachyon_runner.h"
+#include "benchmark/rapidsnark_runner.h"
```

Comment:
It could be kind of minor but they would be better to sort in alphabetical order even in the 'clang-format off'
```suggestion
#include "benchmark/rapidsnark_runner.h"
#include "benchmark/tachyon_runner.h"
```